docs/visualization/index.md

```diff
@@ -1 +1,43 @@
 # Visualization
+
+## Basic APIs Concept
+
+- [Basic APIs Concept](./concept/basic_api_concept.md)
+- [Gallery](../gallery.md)
+
+## CARET APIs
+
+### Visualization APIs
+
+#### Callback
+
+- [callback_sched](./visualization_api/callback_scheduling_visualization.md)
+- [create_callback_frequency_plot](./visualization_api/callback_information.md#execution-frequency)
+- [create_callback_period_plot](./visualization_api/callback_information.md#period)
+- [create_callback_latency_plot](./visualization_api/callback_information.md#latency)
+
+#### Communication
+
+- [create_communication_latency_plot](./visualization_api/communication_information.md#latency)
+- [create_communication_frequency_plot](./visualization_api/communication_information.md#frequency)
+- [create_communication_period_plot](./visualization_api/communication_information.md#period)
+
+#### Node
+
+- [create_publish_subscription_period_plot](./visualization_api/pub_sub_information.md#period)
+- [create_publish_subscription_frequency_plot](./visualization_api/pub_sub_information.md#frequency)
+  <br>Note: Possible to get publisher as the same.
+
+#### Path
+
+- [message_flow](./visualization_api/message_flow.md)
+- [create_response_time_histogram_plot](./visualization_api/response_time.md)
+
+### Investigation APIs
```

Comment:
@keita1523  
Could you tell me why you chose `Investigation`? `Helper` might be better choice for me.

Reply:
@takam5f2 
I will fix this right away. 
Is that ok to change 
> CARET provides some APIs to investigate more detail.  

to  CARET provides some APIs which can help user to get more detail. ?

Reply:
@bopeng-saitama  
I think that `LTTngEventFilter` and `get_callback()` methods are helper utilities.  How about the following sentence?

` CARET provides some APIs which can help user to focus on his/her interest`

Reply:
@takam5f2  I think so too .  So would it be better to change the title of this section to helper utilities?

Reply:
@bopeng-saitama  
`Helper function` or `Utility` might be the appropriate word.